wsl.go

```diff
@@ -391,6 +407,22 @@ func (p *Processor) parseBlockStatements(statements []ast.Stmt) {
 			return false
 		}
 
+		// If it's a short declaration we should not cuddle with anything else
+		// if ForceExclusiveShortDeclarations is set on; either this or the
+		// previous statement could be the short decl, so we'll find out which
+		// it was and use *that* statement's position
+		if p.config.ForceExclusiveShortDeclarations && cuddledWithLastStmt {
+			if p.isShortDecl(stmt) && !p.isShortDecl(previousStatement) {
+				t := stmt.(*ast.AssignStmt)
```

Comment:
Is this needed? I thought you could get `Pos()` from `ast.Stmt` which `stmt` is without casting to a specific `ast` type.

Reply:
Good catch, will commit a fix